### DISCUSSION TITLE
Enforce file, folder naming to be in camelCase

### DISCUSSION THREAD

**Discussion by jatin-vl** [eslint/eslint#17348] [General] [answered]
I used "eslint-plugin-unicorn" for enforcing file naming to be in camelCase in node application. 

It works fine and throw error for the following naming convention.
-  checklist-forecast.ts
-  ChecklistForecast.ts
-  checklist-forecast.ts

But it does not throw error for the "multiplewordstucktogetherwithoutcamelcasing". As it consider it as a single word.
- checklistforecast.ts

Please suggest the possible way to handle this scenario in eslint.

**Comment by nzakas** [comment, marked as answer]
If you have questions about `eslint-plugin-unicorn`, you'll need to ask at that repo. This is just for ESLint itself.
